### DISCUSSION TITLE
Equipo 4 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#10] [Proyecto Final]
Hola @Monolophosaurus 
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by 0Wenn0** [comment]
Hola @Monolophosaurus , soy Wendy, moderadora del equipo de educación.  Escribo para saber si necesitas apoyo con alguna parte del desarrollo de tu proyecto, ¡aún estás a tiempo! Cualquier duda o comentario estoy aquí para ti.

**Reply by Monolophosaurus** [reply]
Hola @0Wenn0, todo bien por ahora. Si necesito algo se lo comunicare sin falta :)

**Comment by Monolophosaurus** [comment]
Hola @KasturiSShah 
Muchas gracias, ya me encuentro trabajando en lo que me falta.
Saludos y nos vemos mañana :)

**Comment by KasturiSShah** [comment]
Hola @Monolophosaurus 

Espero estés muy bien. 

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar su día asignado al otro día.

Estamos (@0Wenn0 y yo) aquí para apoyarte. Cualquier pregunta sobre tu proyecto, déjanos saber por medio de este hilo y te contestaríamos pronto.

Saludos,
Kasturi
